README.md

```diff
@@ -956,8 +956,6 @@ Bookmarks
 
 <dl>
 
-[Easy x86-64](http://ian.seyler.me/easy_x86-64/)
```

Comment:
This can replaced with an archive.org link: https://web.archive.org/web/20190829060538/http://ian.seyler.me/easy_x86-64/